book/website/index.md

```diff
@@ -1,38 +1,38 @@
 (welcome)=
 # Welcome
 
-*Welcome to The Turing Way handbook to reproducible, ethical and collaborative data science.*
+_Welcome to The Turing Way Handbook to Reproducible, Ethical, and Collaborative Data Science._
 
-_The Turing Way_ is an open science, open collaboration, and community-driven project.
-We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
-Our goal is to provide all the information that researchers, data scientists, software engineers, policymakers, and other practitioners in academia, industry, government and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
+The Turing Way is an open science, open collaboration, and community-driven project. 
+We involve and collaborate with a diverse community of contributors to make data science accessible, comprehensible and beneficial for everyone.
 
 ```{admonition} Top Tip
 :class: tip
-_The Turing Way_ is not meant to be read from start to finish.
-Start with a concept, tool or method that you need now, in your current work.
-Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first.
+_The Turing Way_ is not meant to be read from start to finish. 
+Start with what interests you!
+Whether it's a concept, a tool, or a method you need right now, jump in.
+Browse the different guides or use the search box —
+it's all about what you want to learn.
```

Comment:
How do I reject this and go back to what was there?

Reply:
You could suggest an edit which removes all of these lines. Or, if this was a single commit you could revert that commit.